<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RzxJabberPropUI</class>
 <widget class="QWidget" name="RzxJabberPropUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Connection Settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="3" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>642</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Roster Management</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>261</width>
              <height>51</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="rosterEdit" />
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>JID:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="QPushButton" name="rosterDeleteButton" >
              <property name="text" >
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QPushButton" name="rosterAddButton" >
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item rowspan="3" row="0" column="0" >
           <widget class="QListWidget" name="rosterList" />
          </item>
         </layout>
         <widget class="QWidget" name="widget" >
          <property name="geometry" >
           <rect>
            <x>324</x>
            <y>28</y>
            <width>307</width>
            <height>191</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QGroupBox" name="grpReconnect" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>3</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis" >
          <string>&lt;h3>Reconnection&lt;/h3>&lt;p>Defines the parameters of the reconnection.&lt;p>&lt;p>You shouldn't have to modify this values&lt;/p></string>
         </property>
         <property name="title" >
          <string>Reconnection</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>6</number>
          </property>
          <property name="spacing" >
           <number>2</number>
          </property>
          <item row="1" column="0" >
           <widget class="QLabel" name="lblReconnection" >
            <property name="text" >
             <string>Auto-reconnect delay:</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy" >
             <cstring>reconnection</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="lblPingTimeout" >
            <property name="text" >
             <string>Ping timeout (sec):</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy" >
             <cstring>ping_timeout</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="reconnection" >
            <property name="maximum" >
             <number>32768</number>
            </property>
            <property name="minimum" >
             <number>30</number>
            </property>
            <property name="value" >
             <number>60</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSpinBox" name="ping_timeout" >
            <property name="maximum" >
             <number>32768</number>
            </property>
            <property name="value" >
             <number>120</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="grpComm" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>3</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Server identification</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="2" >
           <widget class="QLabel" name="lblServerport" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Port:</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy" >
             <cstring>server_port</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2" >
           <widget class="QLineEdit" name="pass" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis" >
             <string>&lt;h3>Server&lt;/h3>&lt;p>Define the address of the xnetserver&lt;/p></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="cursorPosition" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2" >
           <widget class="QLineEdit" name="user" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis" >
             <string>&lt;h3>Server&lt;/h3>&lt;p>Define the address of the xnetserver&lt;/p></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="cursorPosition" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="lblServername_2_2" >
            <property name="text" >
             <string>Password:</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy" >
             <cstring>server_name</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lblServername_2" >
            <property name="text" >
             <string>User name:</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy" >
             <cstring>server_name</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="lblServername" >
            <property name="text" >
             <string>Server name:</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy" >
             <cstring>server_name</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="QLineEdit" name="server_name" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis" >
             <string>&lt;h3>Server&lt;/h3>&lt;p>Define the address of the xnetserver&lt;/p></string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="cursorPosition" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QSpinBox" name="server_port" >
            <property name="toolTip" >
             <string>&lt;h3>Port&lt;/h3>&lt;p>Defines the port to connect to on the xnetserver&lt;/p>&lt;p>You shouldn't have to modify this value&lt;/p></string>
            </property>
            <property name="maximum" >
             <number>32768</number>
            </property>
            <property name="value" >
             <number>5222</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="lblInfo" >
         <property name="text" >
          <string>Jabber protocole.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>User Info</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="3" column="0" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="1" >
          <widget class="QPushButton" name="getButton" >
           <property name="text" >
            <string>Retrieve</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" >
          <widget class="QPushButton" name="closeButton" >
           <property name="text" >
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" >
          <widget class="QPushButton" name="setButton" >
           <property name="text" >
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="groupBox_3_2" >
         <property name="title" >
          <string>Organisation</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Unit:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Name:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="propOrgUnit" />
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="propOrgName" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>General</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="propBirthday" />
          </item>
          <item row="5" column="0" >
           <widget class="QLabel" name="label_2_2_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Phone:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label_2_2_2_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Website:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLineEdit" name="propWebsite" />
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Full Name:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="propEmail" />
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="propName" />
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="propNick" />
          </item>
          <item row="5" column="1" >
           <widget class="QLineEdit" name="propPhone" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Nickname:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_2_2_2_2_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Birthday:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_2_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Email:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Address</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="4" column="1" >
           <widget class="QLineEdit" name="propCountry" />
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label_3_2_2_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Country:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_3_2_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Post Code:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="propPostCode" />
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="propRegion" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_3_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Region:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>City:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="propCity" />
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="propStreet" />
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Street:&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
